desafio-2/desafio-2.css

```diff
@@ -0,0 +1,32 @@
+body{
+ margin: 0; /* removing the default body margin */
+ text-align: center;
+ box-sizing: border-box
+}
+
+.container{
```

Comment:
for mobile, you can define flex-direction as columns so the divs will be stacked in blocks:

like this: 
```
@media screen and (max-width: 1024px) {   
  .container {
    flex-direction: column;
  }
}
```

the result:
![image](https://user-images.githubusercontent.com/38358239/147238332-dcd99a09-ad6f-41b4-ae00-a2b99191b209.png)

Reply:
Really using `flex-direction: column` is much better for mobile, it looks like this:


<img width="306" alt="Captura de Tela 2021-12-23 às 14 21 06" src="https://user-images.githubusercontent.com/42494406/147273320-cc270093-96f5-4c48-b2c3-8f3e0577d823.png">